.github/workflows/release.yml

```diff
@@ -20,4 +19,8 @@ jobs:
       - name: Release
         run: yarn semantic-release
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.release_bot_github_token}}
+          GIT_AUTHOR_NAME: ${{ secrets.release_bot_git_name }}
+          GIT_AUTHOR_EMAIL: ${{ secrets.release_bot_git_email }}
+          GIT_COMMITTER_NAME: ${{ secrets.release_bot_git_name }}
+          GIT_COMMITTER_EMAIL: ${{ secrets.release_bot_git_email }}
```

Comment:
Documentation supporting this change:

https://github.com/semantic-release/github#github-authentication
https://github.com/semantic-release/git#git-authentication